src/shared/organisms/DataExplorerGraphFlowNavigationStack/NavigationArrows.spec.tsx

```diff
@@ -0,0 +1,238 @@
+          161,
+        ],
+      },
+      {
+        isDownloadable: false,
+        _self:
+          'https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/datashapes:ontologyentity/https:%2F%2Fbbp.epfl.ch%2Fontologies%2Fcore%2Fefeatures%2FNeuroElectroNeuronElectrophysiologicalFeature',
+        title: 'NeuroElectro Neuron Electrophysiological Feature',
+        types: ['Class'],
+        resource: [
+          'neurosciencegraph',
+          'datamodels',
+          'https://bbp.epfl.ch/ontologies/core/efeatures/NeuroElectroNeuronElectrophysiologicalFeature',
+          29,
+        ],
+      },
+      {
+        isDownloadable: false,
+        _self:
+          'https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/_/https:%2F%2Fneuroshapes.org',
+        title: 'neuroshapes.org',
+        types: [],
+        resource: [
+          'neurosciencegraph',
+          'datamodels',
+          'https://neuroshapes.org',
+          161,
+        ],
+      },
+    ],
+    shrinked: false,
+  },
+  limited: false,
+  referer: {
+    pathname: '/my-data',
+    search: '',
+    state: {},
+  },
+};
+
+describe('NavigationStack', () => {
+  let app: JSX.Element;
+  let component: RenderResult;
+  let container: HTMLElement;
+  let rerender: (ui: React.ReactElement) => void;
+  let store: Store<any, AnyAction>;
+  let user: UserEvent;
+  let history: MemoryHistory<{}>;
+  let nexus: NexusClient;
+
+  beforeEach(() => {
+    history = createMemoryHistory({});
+
+    nexus = createNexusClient({
+      fetch,
+      uri: deltaPath(),
+    });
+    store = configureStore(history, { nexus }, {});
+    app = (
+      <Provider store={store}>
+        <Router history={history}>
+          <NexusProvider nexusClient={nexus}>
+            <>
+              <NavigationArrows />
+              <NavigationStack key="navigation-stack-left" side="left" />
+              <NavigationStack key="navigation-stack-right" side="right" />
+            </>
+          </NexusProvider>
+        </Router>
+      </Provider>
+    );
+    component = render(app);
+    container = component.container;
+    rerender = component.rerender;
+    user = userEvent.setup();
+  });
+
+  it('should render the back/forward arrows as not disabled', () => {
+    store.dispatch(
+      ResetDataExplorerGraphFlow({ initialState: initialDataExplorerState })
+    );
+    rerender(app);
+
+    const backArrow = container.querySelector('[aria-label="back-arrow"]');
+    const forwardArrow = container.querySelector(
+      '[aria-label="forward-arrow"]'
+    );
+    expect(backArrow).toBeInTheDocument();
+    expect(forwardArrow).toBeInTheDocument();
+  });
+  it('should left side of navigation become 3 and right side become 2 when Forward btn clicked', async () => {
+    store.dispatch(
+      ResetDataExplorerGraphFlow({ initialState: initialDataExplorerState })
+    );
+    rerender(app);
+    const forwardArrow = container.querySelector(
```

Comment:
Since the first two lines are in all specs, they can be added to the `beforeEach` hook:

```
   store.dispatch(
      ResetDataExplorerGraphFlow({ initialState: initialDataExplorerState })
    );
    rerender(app);
```